substrate/frame/src/lib.rs

```diff
@@ -204,7 +204,10 @@ pub mod prelude {
 	#[doc(no_inline)]
 	pub use frame_support::dispatch::{GetDispatchInfo, PostDispatchInfo};
 	pub use frame_support::traits::{
-		Contains, EstimateNextSessionRotation, IsSubType, OnRuntimeUpgrade, OneSessionHandler,
+		fungible::hold::{Inspect, Mutate},
```

Comment:
I don't think we should export those here, they will conflict with `fungibles::hold`.
I think we should just reexport `fungible` and `fungibles`.